scripts/generate-dapps.ts

```diff
@@ -18,6 +20,10 @@ const HEADER_CONTENT = `// =====================================================
 import { type Dapp } from '../types';
 `;
 
+function toImprovisedKebabCase(value: string) {
```

Comment:
There is this in the same repo https://github.com/api3dao/contracts/blob/main/src/cli.ts#L40

Reply:
I'll copy paste it, because I don't want to create an utils module just for this function.